### PR TITLE
Cleanup favorites from mvc to designsystem

### PR DESCRIPTION
**Pull request by magnusnn**
Changes:
* Toggle star on hover if not mobile
* Changed color and size of star
* Fixed padding and margin around button
* Updated tags and mustache

### CODE REVIEW

**Comment by magnusnn** [comment]
har dere mulighet til å se på denne @clarkt @matsgm

File: package.json [hunk 1]
```diff
@@ -61,14 +61,14 @@
     "jquery-validation-unobtrusive": "3.2.6",
     "mark.js": "8.11.1",
     "minimist": "1.2.0",
-    "patternlab-node": "2.9.3",
+    "patternlab-node": "^2.12.0",
```
Comment (matsgm) [inline package.json]: Please use exact versioning for all modules in package.json

Comment (matsgm) [reply package.json]: Also when updating package.json I would guess that the package-lock.json also would be updated?

Comment (magnusnn) [reply package.json]: reverted the version-changes

**Pull request merged**